test/unit/enrichment/process-invoice-lines.test.js

```diff
@@ -8,12 +8,15 @@ const { NET_DESCRIPTION, GROSS_DESCRIPTION } = require('../../mocks/values/descr
 const { SOURCE_SYSTEM } = require('../../mocks/values/source-system')
 const scheme = require('../../mocks/scheme')
 const invoiceLine = require('../../mocks/payment-requests/invoice-line')
+
 const invoiceLines = [
   invoiceLine,
   invoiceLine,
   { ...invoiceLine, description: NET_DESCRIPTION }
 ]
 
+const marketingYear = 2023
+
```

Comment:
I think these should be below processInvoiceLines

Reply:
Also moved.